### ISSUE TITLE
Fix gulp deploy so it doesn't simply delete origins/gh-pages

### ISSUE THREAD

**Issue by axerosh** [holmgr/gamejam#14]
After gulp deploy is finished, the gh-pages branch at origins has been deleted. The terminal output seems fine:

```
[00:21:43] Starting 'deploy'...
[00:21:43] Temporarily committing dist
[00:21:45] Cleaning origin/gh-pages
[00:21:49] Pushing dist to origin/gh-pages
[00:21:49] Resetting dist temporary commit
[00:21:49] Finished 'deploy' after 5.92 s
```

**Comment by axerosh** [comment]
Fixed in c8a609b

**Comment by axerosh** [comment]
Fixed in f05d70a1ee9dd90f05d085b139702ac0b4346848
